pom.xml

```diff
@@ -74,7 +74,7 @@
             <plugin>
                 <groupId>io.openliberty.tools</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
-                <version>3.3-M4</version>
+                <version>3.3</version>
```

Comment:
Use 3.3.1 (although it doesn't affect this project, that is the latest version which has an unrelated fix in a parent pom)